### DISCUSSION TITLE
Remapping a Sunshine's complexed shortcut to another simpler shortcut?

### DISCUSSION THREAD

**Discussion by Barina** [LizardByte/community#387] [Sunshine]
Hi all!

Let me describe my issue.
My host setup runs Win 11, latest Sunshine and has 2 monitors attached. sometimes there's a need to change to the second monitor. (f.ex when UAC prompts or the game insists running on specific monitor etc.)
On my "client" setup, I'm using a mini PC with LMDE6, latest Moonlight and a [Rii Mini Keyboard i4](http://www.riitek.com/product/214.html).
This keyboard requires pressing the `FN` button to access the F1-12 keys..

There's some cool Sunshine shortcuts that let me switch between monitors like Ctrl+Alt+Shift+F1-F3
You probably already see the issue.. with my keyboard I need to press 5 different buttons.. it is very clunky and doesn't even work (sometimes?)
I can't find any way of assigning more reasonable shortcuts to Sunshine's actions such as described above.. so I thought maybe I can just remap these shortcuts into another, more simple shortcuts..

Any ideas?

I tried playing with xbindkeys but this is more to launch commands.. I thought maybe I can run a command that sends a shortcut?

**Comment by elijabesu** [comment]
Has anyone figured anything out? I'm using Sunshine to stream to my iPad where I can't press F keys, so I'm just stuck to the main monitor only ;_;

**Comment by slrgt** [comment]
would love to be able to do this so I can enable Mouse Mode on my RG35XXH for the few times i need to move or click on the mouse

**Comment by XPRAMT** [comment]
https://github.com/Axixi2233/moonlight-android
![Screenshot_20241008-043941](https://github.com/user-attachments/assets/1d42c0de-8cb6-4164-8970-94ac640af1bc)

[a comment by an automated account is not shown]
